cpp/src/arrow/compute/api_scalar.h

```diff
@@ -48,6 +48,26 @@ namespace compute {
 ARROW_EXPORT
 Result<Datum> Add(const Datum& left, const Datum& right, ExecContext* ctx = NULLPTR);
 
+/// \brief Subtract two values. Array values must be the same length. If a
```

Comment:
Can you write out the names of these functions, `Subtract` and `Multiply`?

Reply:
Sure.